### DISCUSSION TITLE
Add sensor.octopus_energy_electricity_[id]_export_current_accumulative_cost [Home Mini]

### DISCUSSION THREAD

**Discussion by Leventionz** [BottlecapDave/HomeAssistant-OctopusEnergy#395] [Ideas]
Hi Dave,

I use a home mini and would like to see the current accumulative export profit (cost), currently I can see the previous period (yesterday); are you able to add this sensor for home mini users? e.g:-

sensor.octopus_energy_electricity_[id]_export_current_accumulative_cost

Thank you very much and keep up the excellent work.

**Comment by BottlecapDave** [comment]
Hi there.

Unfortunately, the last time I checked the Home Mini doesn't report on export data, which is why there are no current consumption sensors for export meters. Are you able to see export data within the app?

**Reply by Leventionz** [reply]
Apologies for the belated reply, no unfortunately it is not visible within the app which is frustrating.

I have managed a kind of workaround now by taking the current export price from your integration and combining that with the current sensor I have on my inverter to give me an approximate export value; it doesn't match exactly but gives me a rough estimate.

Thanks for the quick response btw and apologies again for the late reply.
